recipes/nrtk-jatic/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nrtk-jatic" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_jatic-{{ version }}.tar.gz
+  sha256: 57d827c541d9b26a1df464516a79a3b3603eb0d8302f5b68b9b6606a7cf52bfe
+
+build:
+  skip: true  # [py>=400 or py2k]
+  skip: true  # [py==38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry-core
+  run:
+    - python
+    - numpy <1.26,>=1.22  # [py>=39 and py<312]
```

Comment:
What's the reason for this numpy pin? Without it, this could be a noarch recipe.

Reply:
1. skip: true  # [py==38] is due to the nrtk dependency (https://github.com/conda-forge/nrtk-feedstock) (most recent versions) building successfully only for py>=39
2.  https://github.com/Kitware/nrtk-jatic/blob/ba40077ef165480f1867b2649c612da2e5682dba/pyproject.toml#L37
Here, the constrained dependencies are captured. I tried without the numpy pin and the conda resolver fails to resolve.